vendor/github.com/mitchellh/cli/cli.go

```diff
@@ -578,6 +578,12 @@ func (c *CLI) processArgs() {
 			break
 		}
 
+		// Check for help flags.
+		if arg == "-h" || arg == "-help" || arg == "--help" {
```

Comment:
Should this be in this PR or is it a no-op with the de-monkey-patch change that just got merged?

Reply:
This is the reversal of the monkey patch and it should be in this PR. The other de-monkey-patch was for mitchellh/mapstructure not mitchellh/cli.

Reply:
Got it - ok.